src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/mailer.html.twig

```diff
@@ -187,6 +187,14 @@
                                                     <div class="tab">
                                                         <h3 class="tab-title">Attachment #{{ loop.index }}</h3>
                                                         <div class="tab-content">
+                                                            {% set filename = attachment.filename|default() %} {# symfony/mime < 6.1 would resolve to null #}
+
+                                                            {% if filename %}
+                                                                <p>
+                                                                    <a href="data:{{ attachment.contentType }};base64,{{ collector.base64Encode(attachment.body) }}"
```

Comment:
maybe for <6.1 octet-stream generally works as default content-type :grimacing:

Reply:
`application/octet-stream` is already the default content-type as determined in `DataPart::__construct()`